taichi/backends/vulkan/runtime.cpp

```diff
@@ -442,26 +467,41 @@ class VkRuntime ::Impl {
     // Need to zero fill the buffers, otherwise there could be NaN.
     Stream *stream = device_->get_compute_stream();
     auto cmdlist = stream->new_command_list();
-    cmdlist->buffer_fill(root_buffer_->get_ptr(0), root_buffer_size,
-                         /*data=*/0);
+
     cmdlist->buffer_fill(global_tmps_buffer_->get_ptr(0), gtmp_buffer_size,
                          /*data=*/0);
     stream->submit_synced(cmdlist.get());
   }
 
-  const SNodeDescriptorsMap *const snode_descriptors_;
+  void add_root_buffer() {
+#pragma message("Vulkan buffers size hardcoded")
+    size_t root_buffer_size = 64 * 1024 * 1024;
```

Comment:
After materializing the SNode tree, we should be able to get a size for that tree?

Reply:
Since we are not handling sparse yet

Reply:
hmm yeah. lemme try.